cmds/tui.c

```diff
@@ -252,7 +249,34 @@ static void print_graph_total(struct field_data *fd)
 	struct uftrace_graph_node *node = fd->arg;
 	uint64_t d;
 
-	d = node->time;
+	d = node->total_time.sum;
+
+	print_time(d);
+}
+
+static void print_graph_total_avg(struct field_data *fd)
+{
+	struct uftrace_graph_node *node = fd->arg;
+	uint64_t d;
+	d = node->total_time.avg;
```

Comment:
Please have a blank line after the variable declaration (here and in other functions).